docs/en/integrations/data-ingestion/apache-spark/spark-native-connector.md

```diff
@@ -0,0 +1,583 @@
+---
+sidebar_label: Spark Native Connector
+sidebar_position: 2
+slug: /en/integrations/apache-spark/spark-native-connector
+description: Introduction to Apache Spark with ClickHouse
+keywords: [ clickhouse, apache, spark, migrating, data ]
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import TOCInline from '@theme/TOCInline';
+
+# Spark Connector
+
+This connector leverages ClickHouse-specific optimizations, such as advanced partitioning and predicate pushdown, to
+improve query performance and data handling.
+The connector is based on [ClickHouse's official JDBC connector](https://github.com/ClickHouse/clickhouse-java), and
+manages its own catalog.
+
+Before Spark 3.0, Spark lacked a built-in catalog concept, so users typically relied on external catalog systems such as Hive Metastore or AWS Glue.
+With these external solutions, users had to register their data source tables manually before accessing them in Spark.
+However, since Spark 3.0 introduced the catalog concept, Spark can now automatically discover tables by registering catalog plugins.
+
+Spark default catalog is `spark_catalog`, and tables are identified by `{catalog name}.{database}.{table}`. With the new
+catalog feature, it is now possible to add and work with multiple catalogs in a single Spark application.
+
+
+<TOCInline toc={toc}></TOCInline>
+
+## Requirements
+
+- Java 8 or 17
+- Scala 2.12 or 2.13
+- Apache Spark 3.3 or 3.4 or 3.5
+
+## Compatibility Matrix
+
+| Version | Compatible Spark Versions | ClickHouse JDBC version |
+|---------|---------------------------|-------------------------|
+| main    | Spark 3.3, 3.4, 3.5       | 0.6.3                   |
+| 0.8.0   | Spark 3.3, 3.4, 3.5       | 0.6.3                   |
```

Comment:
We need to update the table. We already have 0.8.1. I will update the repository since we did not mention it there.

Reply:
| 0.8.1   | Spark 3.3, 3.4, 3.5       | 0.6.3                   |